Dockerfile.development

```diff
@@ -0,0 +1,18 @@
+FROM starefossen/ruby-node:2-8
+
+ENV APP_ROOT /champaign
+
+RUN mkdir $APP_ROOT
+WORKDIR $APP_ROOT
+
+COPY Gemfile Gemfile.lock ./
+RUN bundle install --jobs 20 --retry 3
+
+COPY package.json yarn.lock $APP_ROOT/
+RUN yarn
+
+COPY . $APP_ROOT/
+
+EXPOSE 3000 8080
+
+CMD bundle exec foreman start
```

Comment:
I’ve been running into troubles every time I restarted the containers. It was terminating after Rails process sent a `SIGTERM`. I had difficulties to find out why, since I could start and restart `bundle exec foreman start` at will from a `docker-compose run web bash` container. Eventually, I’ve figured out the container wasn’t terminating properly and didn’t reap orphaned zombie processes (Foreman and Puma for instance). 

This is a [known](https://engineeringblog.yelp.com/2016/01/dumb-init-an-init-for-docker.html) [issue](https://blog.phusion.nl/2015/01/20/docker-and-the-pid-1-zombie-reaping-problem/) that can be easily addressed with [dumb-init](https://github.com/Yelp/dumb-init) (a minimalist init system):

```
RUN wget -O /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64
RUN chmod +x /usr/local/bin/dumb-init

ENTRYPOINT ["/usr/local/bin/dumb-init", "--"]
CMD ["bundle", "exec", "foreman", "start"]
```